build/style_test.go

```diff
@@ -706,7 +687,10 @@ func TestStyle(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		if err := stream.ForEach(filter, func(s string) {
+		if err := stream.ForEach(stream.Sequence(
+			filter,
+			stream.GrepNot(`: (field no|type No)Copy is unused \(U1000\)$`),
```

Comment:
Yes, that's true (in fact, this PR already uses that flag to ignore some other files). However, in this case each use of this type (not just its declaration) results in a warning, and ignoring all those files is not something we want to do.